pkg/controller/publishingstrategy/publishingstrategy_controller.go

```diff
@@ -370,18 +377,29 @@ func (r *ReconcilePublishingStrategy) deleteIngressWithAnnotation(appIngressList
 }
 
 // contains check if an individual non-default ingress on cluster matches with any non-default applicationingress
+// return true if ingress (on cluster) matches with any applicationIngress in PublishingStrategy CR
+// return false if ingress (on cluster) DOES NOT match with ANY applicationIngress in PublishingStrategy CR
+// this helps to determine if ingress needs to be deleted or not. The end goal is to have all ApplicationIngress on PublishingStrategy CR on cluster
 func contains(appIngressList []cloudingressv1alpha1.ApplicationIngress, ingressController *operatorv1.IngressController) bool {
+	var isContained bool
 	for _, app := range appIngressList {
+		log.Info(fmt.Sprintf("app being processed %s", app.DNSName))
+		// TODO (sam): Add a comment about why this if is here, and the above method too
+		// if the ApplicationIngress (on CR) is default then set bool to false
+		// eg. ingresscontroller (on cluster): apps2
+		//     appIngressList (in CR) : [default]
+		// since apps2 does NOT exist in appIngressList, set bool to false and ready for deletion
```

Comment:
```suggestion
		// if the ApplicationIngress (on CR) is default then set bool to false
		// eg. ingresscontroller (on cluster): apps2
		//     appIngressList (in CR) : [default]
		// since apps2 does NOT exist in appIngressList, set bool to false and ready for deletion
```

Looks like you wrote your comment? ;)